pool.go

```
@@ -106,30 +105,36 @@ func (p *Pool) Close() {
 		if client.ClientConn == nil {
 			continue
 		}
-		client.ClientConn.Close()
+		_ = client.ClientConn.Close()
```

You don't need to assign the result of `Close()` to `_`.

---

I use `gometalinter` to check my code. So I used _ to ignore the error. It's my habit....